platformcomponents/desktop/content-file-card-item.json

```diff
@@ -14,6 +14,10 @@
       "primary": "@theme-text-primary-normal",
       "secondary": "@theme-text-secondary-normal",
       "disabled": "@theme-text-primary-disabled"
+    },
```

Comment:
what is ecmBox? do you mean the label white box in the upper corner? in case you do, there is a component already for this under platformcomponents/tags.